composer.json

```diff
@@ -7,9 +7,9 @@
     ],
     "license": "MIT",
     "require": {
-        "php": ">=7.1.0",
+        "php": ">=8.1",
```

Comment:
Any reason to drop support for php 7?

Reply:
Can you confirm the composer require definition in LE for this package for me? Just want to make sure it's using a tagged release and not a branch name.

Reply:
It's using `1.0.3-rc3`, so we can safely merge this in and make a new release for the PHP upgrade.